app/src/main/res/values/strings.xml

```diff
@@ -135,26 +135,26 @@
     <string name="number_of_files" needed="false">Number of files: %d</string>
 
     <!-- New strings -->
-    <string name="global">Global</string>
+    <string name="global">Program-wide</string>
```

Comment:
What does "global" mean here? It is very easy to get wrong in translation by assuming its meaning.

Reply:
where do you have this string out of? it is currently not more in the list to translate. do you have an old git version?

Reply:
App wide